src/index.ts

```diff
@@ -1 +1,2 @@
-export * from "./MicrosoftTeams";
+export * from "./private/index";
+export * from "./public/index";
```

Comment:
Did you figure out how to generate a single .d.ts file for the SDK after these changes or are we deciding that's no longer necessary?

Reply:
We can do it if partner can not consume npm package for typing.